cmd/notation/plugin.go

```diff
@@ -36,6 +44,44 @@ Example - List installed Notation plugins:
 	}
 }
 
+func pluginInstallCommand() *cobra.Command {
+	var force bool
+
+	cmd := &cobra.Command{
+		Use:     "install [flags] <plugin package>",
+		Aliases: []string{"add"},
+		Short:   "Install a plugin",
+		Long: `Install a plugin
+
+		Example - Install a Notation plugin:
+			notation plugin install <plugin package>
+`,
```

Comment:
Indentation should be removed. The output should not follow the code indentitation.
```suggestion
		Long: `Install a plugin

Example - Install a Notation plugin:
	notation plugin install <plugin package>
`,
```